napptive/oam-kubernetes-runtime
# Fix/pg 552 serveral ports

### Checklist

#### What does this PR do?
Adds all the ports defined in WorkloadDefinition to the service generated

#### Where should the reviewer start?

#### What is missing?

#### How should this be manually tested?
Deploy an application that has several ports defined in a workload. Get the service and check the ports defined

#### Any background context you want to provide?

#### What are the associated tickets?

#### Screenshots (if appropriate)

#### Questions
